pkgmgr/package.go

```diff
@@ -43,6 +43,10 @@ func (p Package) install(cfg Config, context string) (string, error) {
 		cfg.DataDir,
 		pkgName,
 	)
+	pkgConfigDir := filepath.Join(
+		cfg.ConfigDir,
+		pkgName,
+	)
```

Comment:
I don't think we want to use `ConfigDir` for anything other than the actual `cardano-up` config. All package data should stay in the cache/data dirs.

Reply:
How do I get `config` for "--node-config", "./config/mainnet/config.json"?

Reply:
They're already in the container.

Older images were network-specific:
```$ docker run --rm -ti --entrypoint sh cardanosolutions/ogmios:v5.6.0-mainnet -c 'ls -al /config/cardano-node'
total 16
drwxr-xr-x    2 root     root          4096 Aug 12  2023 .
drwxr-xr-x    5 root     root          4096 Aug 12  2023 ..
-rw-r--r--    1 root     root          2725 Aug 12  2023 config.json
-rw-r--r--    1 root     root           128 Aug 12  2023 topology.json
```

While newer images are network agnostic:
```
$ docker run --rm -ti --entrypoint sh cardanosolutions/ogmios:v6.1.0 -c 'ls -al /config/mainnet/cardano-node'
total 16
drwxr-xr-x    2 root     root          4096 Feb 21 10:57 .
drwxr-xr-x    6 root     root          4096 Feb 21 10:57 ..
-rw-r--r--    1 root     root          3006 Feb 21 10:57 config.json
-rw-r--r--    1 root     root           529 Feb 21 10:57 topology.json
```